internal/config/src/jest/index.js

```diff
@@ -0,0 +1,18 @@
+const getPackages = require('get-monorepo-packages')
+
```

Comment:
This module allows  ts-jest to pick up changes in other packages without having to run build on every external change.